modules/tile-converter/package.json

```diff
@@ -67,6 +67,9 @@
     "morgan": "~1.9.1",
     "uuid": "^8.1.0"
   },
+  "devDependencies": {
```

Comment:
Why a dev dependency?

Reply:
My idea was that I used only one type from `@loaders.gl/schema`. Should I move it to "dependencies"?

Reply:
Well, if the tile-converter is a binary, not published as an importable npm module it may not matter.

So when someone installs it it is fine.

But if someone imports and uses typescript it I suppose they will need all dependencies that are actually imported?